doc/nrf/links.txt

```diff
@@ -659,7 +659,6 @@
 
 .. _`CONFIG_BT_CTLR_TX_PWR_MINUS`: https://docs.nordicsemi.com/bundle/ncs-latest/page/kconfig/index.html#!CONFIG_BT_CTLR_TX_PWR_MINUS
 .. _`CONFIG_BT_CTLR_TX_PWR_PLUS`: https://docs.nordicsemi.com/bundle/ncs-latest/page/kconfig/index.html#!CONFIG_BT_CTLR_TX_PWR_PLUS
-.. _`PSA CRYPTO SSF SERVICE ENABLED`: https://docs.nordicsemi.com/bundle/ncs-latest/page/kconfig/index.html#!CONFIG_SSF_PSA_CRYPTO_SERVICE_ENABLED
```

Comment:
This link could be removed from the "C:\ncs\nrf\samples\crypto\psa_tls\README.rst" sample doc. This fixes the doc build error.

Reply:
Removing the link from the sample doc would leave the sample with an undocumented backend I think, so I updated the reference instead of removing it.